pkg/reporter/reporter.go

```diff
@@ -0,0 +1,106 @@
+package reporter
+
+import (
+	"github.com/codefresh-io/cli-v2/pkg/log"
+	"github.com/codefresh-io/cli-v2/pkg/store"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
```

Comment:
I would use `var G = Get` here, just because it's cooler to use `ar.G().ReportStep...` instead of `ar.Get()...`